runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -32,18 +32,18 @@
 import java.util.Set;
 import javax.annotation.Nullable;
 import javax.inject.Inject;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.ScriptID;
-import net.runelite.api.Skill;
-import net.runelite.api.VarPlayer;
-import net.runelite.api.Varbits;
+
+import net.runelite.api.*;
```

Comment:
Please reverse these wildcard import and extra newline changes. [Per our Code Conventions](https://github.com/runelite/runelite/wiki/Code-Conventions#imports) we generally never use them for non-static imports, and even for static imports don't use them frequently.